snapshots/wb/2023-09-19/us_cpi.py

```diff
@@ -1,17 +1,33 @@
+"""Script to create a snapshot of dataset."""
+
 import xml.etree.ElementTree as ET
+from pathlib import Path
 
+import click
 import pandas as pd
 import requests
-from structlog import get_logger
+from owid.datautils.io import df_to_file
+
+from etl.snapshot import Snapshot
 
-log = get_logger()
+# Version for current snapshot dataset.
+SNAPSHOT_VERSION = Path(__file__).parent.name
 
-# change number of years to go back if neccessary (used up to 2000 in 2023)
 
-YEARS = 23
+@click.command()
+@click.option("--upload/--skip-upload", default=True, type=bool, help="Upload dataset to Snapshot")
+def main(upload: bool) -> None:
+    # Create a new snapshot.
+    snap = Snapshot(f"wb/{SNAPSHOT_VERSION}/us_cpi.csv")
+    url = snap.metadata.origin.url_download
```

Comment:
If you add `  # type: ignore` at the end of the line, the error will go away. (Don't be afraid to use it a lot, it's much faster than properly fixing all those type errors)